src/types/markers.js

```diff
@@ -543,12 +543,9 @@ export type LogMarkerPayload = {|
 |};
 
 export type DOMEventMarkerPayload = {|
-  type: 'tracing',
-  category: 'DOMEvent',
-  timeStamp?: Milliseconds,
-  interval: 'start' | 'end',
+  type: 'DOMEvent',
+  latency?: Milliseconds,
```

Comment:
I'm not sure if I understand this commit ("Change the DOMEvent marker to no longer be a tracing marker"). I see that we change the payload type and removed/added some fields. But where is the place we are actually processing these markers? Does that mean that it's gonna be send directly from gecko like this or do we need to process the gecko marker? If we are changing the gecko, can you give me the bug link?

Edit: oh I saw the upgrader now. So looks like we are changing the gecko side.